public/modules/custom/grants_industries/grants_industries.module

```diff
@@ -227,6 +226,16 @@ function grants_industries_openid_connect_post_authorize(UserInterface $account,
       if (str_contains($ad_group_lower, 'ta_sote')) {
         $industryKey = 'SOTE';
       }
+      // New mappings.
+      if (str_contains($ad_group_lower, 'laaja_Kanslia') || str_contains($ad_group_lower, 'suppea_Kanslia')) {
```

Comment:
Nää verrattavat stringit pitäis varmaan kans olla lower casella että mätsää?

Reply:
Hyvä spotti! Korjasin.